src/app/news/news-list-table-rows/news-list-table-rows.component.html

```diff
@@ -6,7 +6,7 @@
     <span>{{activity.project?.name}}</span>
     <br>
     <br>
-    <span [innerHtml]="activity.content"></span>
+    <span class="ck-content" [innerHtml]="domSanitizer.bypassSecurityTrustHtml(activity.content)"></span>
```

Comment:
Just making sure that this was the last resort for this. https://angular.io/api/platform-browser/DomSanitizer
There was no other way to allow iframe only or select set of html?